app/src/androidTest/java/com/stl/skipthelibrary/US010101Test.java

```diff
@@ -0,0 +1,69 @@
+package com.stl.skipthelibrary;
+
+import android.app.Activity;
+import android.view.View;
+import android.widget.EditText;
+
+import com.robotium.solo.Solo;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.runner.AndroidJUnit4;
+
+import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+@RunWith(AndroidJUnit4.class)
+public class US010101Test extends ActivityTestRule<MyBooksActivity>{
+
+    private Solo solo;
+
+    public US010101Test() {
+        super(MyBooksActivity.class, false, true);
```

Comment:
@gillianpierce There a reason you have touch mode off here? I just ask cause pretty sure we had it on in the lab

Reply:
Nah dawg I copied that part exactly from the finalized lab code github repo

Reply:
buddy did it different in lab i swear

Reply:
whatever this is probably good, just keep on going @gillianpierce